user_manual/webinterface.rst

```diff
@@ -11,31 +11,29 @@ your Nextcloud server URL (e.g. cloud.example.com) and enter your username and p
 Web browser requirements
 ------------------------
 
-For the best experience with the Nextcloud web interface, we recommend that you use the 
-latest and supported version of a browser from this list:
-
+For the best experience with the Nextcloud web interface, we recommend that
+you use the latest and supported version of a browser from this list:
 
 .. No need to translate
-
-* Google **Chrome**/Chromium
+* Google **Chrome**/Chromium (Desktop and Android)
```

Comment:
Still no need to translate if we add 'and' ?
Same for others

Reply:
https://www.transifex.com/nextcloud/nextcloud-user-documentation/translate/#de/$/184482352

Doesn't seem to work anyway